examples/src/main/java/io/dapr/examples/tracing/README.md

```diff
@@ -0,0 +1,200 @@
+    // Simulate slow processing for metrics.
+    Thread.sleep(1000);
+  }
+  //...
+}
+```
+
+The instrumentation for the service happens via the `OpenTelemetryIterceptor` class. This class uses the [OpenTelemetrySDK](https://github.com/open-telemetry/opentelemetry-java) for Java.
+
+Use the follow command to execute the service:
+
+```sh
+dapr run --components-path ./components --app-id tracingdemo --app-port 3000 -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.tracing.TracingDemoService -p 3000
+```
+
+Once running, the TracingDemoService is now ready to be invoked by Dapr.
+
+
+### Running the InvokeClient sample
+
+This sample code uses the Dapr SDK for invoking two remote methods (`echo` and `sleep`). It is also instrumented with OpenTelemetry. See the code snippet below:
+
+```java
+public class InvokeClient {
+
+private static final String SERVICE_APP_ID = "invokedemo";
+///...
+  public static void main(String[] args) throws IOException {
+    Tracer tracer = OpenTelemetryConfig.createTracer(InvokeClient.class.getCanonicalName());
+
+    Span span = tracer.spanBuilder("Example's Main").setSpanKind(Span.Kind.CLIENT).startSpan();
+    try (DaprClient client = (new DaprClientBuilder()).build()) {
+      for (String message : args) {
+        try (Scope scope = tracer.withSpan(span)) {
+          InvokeServiceRequestBuilder builder = new InvokeServiceRequestBuilder(SERVICE_APP_ID, "echo");
+          InvokeServiceRequest request
+              = builder.withBody(message).withHttpExtension(HttpExtension.POST).withContext(Context.current()).build();
+          client.invokeService(request, TypeRef.get(byte[].class))
+              .map(r -> {
+                System.out.println(new String(r.getObject()));
+                return r;
+              })
+              .flatMap(r -> {
+                InvokeServiceRequest sleepRequest = new InvokeServiceRequestBuilder(SERVICE_APP_ID, "sleep")
+                    .withHttpExtension(HttpExtension.POST)
+                    .withContext(r.getContext()).build();
+                return client.invokeService(sleepRequest, TypeRef.get(Void.class));
+              }).block();
+        }
+      }
+
+      // This is an example, so for simplicity we are just exiting here.
+      // Normally a dapr app would be a web service and not exit main.
+      System.out.println("Done");
+    }
+    span.end();
+    shutdown();
+  }
+///...
+}
+```
+
+The class knows the app id for the remote application. It uses `invokeService` method to invoke API calls on the service endpoint. The request object includes an instance of `io.grpc.Context` for the proper tracing headers to be propagated.
+ 
+Execute the follow script in order to run the InvokeClient example, passing two messages for the remote method:
+```sh
+dapr run --components-path ./components -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.tracing.InvokeClient 'message one' 'message two'
```

Comment:
for Windows, message one and message two should be wrapped by double quotes instead.

Reply:
done.